src/lib/plugin.js

```diff
@@ -379,10 +379,11 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' is not mandatory
-    ['currency_code', 'scale', 'connectors'].map(field => {
-      if (!res.body[field]) { throwErr() }
-    })
+    // the 'ilp_prefix' field is not mandatory
+    const fields = Object.keys(res.body)
+    if (fields.indexOf('currency_code') === -1) { throwErr() }
```

Comment:
maybe one of the fields is equal to zero? you could check for equality to undefined instead